app/components/rdf-input-fields/mandataris-fractie-selector.hbs

```diff
@@ -15,6 +15,7 @@
         @onClose={{fn (mut this.hasBeenFocused) true}}
         @person={{this.person}}
         @isInCreatingForm={{this.isPersonInForm}}
+        @allowClear={{not this.isRequiredForBestuursorgaan}}
```

Comment:
Should this not be the `@isRequired`, the argument `@allowClear` is no longer used in the fractie-selector component.

Reply:
It should be possible to allow clear when the fractie selector is not required IMO so I forgot to change the false to not isFractieSelector required (previous comment)